src/core/background/pages.js

```diff
@@ -24,7 +24,7 @@ const getTimedItems = scene => {
     return timedItems.map(createItems(scene));
 };
 
-const skip = timedItems => timedItems.forEach(item => item.stop(1));
+export const skip = timedItems => timedItems.forEach(item => item.stop(1));
```

Comment:
what does the 1 do?

Reply:
Tween stop(1) skips to the end of the tween.